packages/cli/generators/openapi/templates/src/models/type-template.ts.ejs

```diff
@@ -0,0 +1,14 @@
+<%_
+const _referencedModels = locals.referencedModels || [];
+const _properties = locals.properties || [];
+_referencedModels.forEach(m => {
+%>
+import {<%- m.className %>} from './<%- m.fileName.substring(0, m.fileName.length -3) %>';
+<%_
+});
+-%>
+/**
+ * The model type is generated from OpenAPI schema - <%- name %>
```

Comment:
What's the different between `model-template` and `type-template`? I find "model type" slightly confusing, because to me, "model type" is a synonym for "model class".

Reply:
> What's the different between `model-template` and `type-template`? I find "model type" slightly confusing, because to me, "model type" is a synonym for "model class". 

@raymondfeng ☝️

Reply:
I think `model-template` doesn't contain property `id`, the model type is an intersection of model class and id

Reply:
I use `type` for TypeScript type declaration/alias and `model` for class.